### ISSUE TITLE
Webcontext can start with `/`

### ISSUE THREAD

**Issue by anilgursel** [paypal/squbs#340]
The doc currently reads:

> The web-context is a string that uniquely identifies the web context of a request to be dispatched to this service. It **MUST NOT** start with a `/` character. It can have `/` characters inside as segment separators in case of multi-segment contexts. And it is allowed to be `""` for root context. If multiple services match the request, the longest context match takes precedence.

We should actually allow it to start with a `/`.  Otherwise, it might get annoying while troubleshooting in development.

**Comment by crankydillo** [comment]
Yes, this just bit me for about 30 minutes.  I figured it out by changing the value until it worked.  However, I did want to find out if it was documented.  The inability to search the docs made that painful.  I did find [the relevant doc](https://github.com/paypal/squbs/blob/b88a218aa0c1daea05d50211f7d1803712170a9c/docs/http-services.md#the-web-context) by doing a github search.

If you really don't want, I'd consider throwing an exception or logging an error/warning.

**Comment by anilgursel** [comment]
I agree, we should either throw an exception or should support `/`.  We need to re-enable search, please see https://github.com/paypal/squbs/issues/534.
